.pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+gulp unit-test
```

Comment:
`gulp unit-test` depends on `jshint`, so it will run jshint before running `unit-test`